scenarios/README.md

```diff
@@ -0,0 +1,35 @@
+# What are Spark .NET Scenarios?
+
+Scenarios are intended as a one-stop location in the repo for understanding 
+various combinations of technology stacks and putting together a working 
+end-to-end Spark .NET app.
+
+# Pre-requisites
+
+Some pre-requisites before you plan on contributing a scenario:
+  1. Datasets you are using in the build scenario are public
+  2. Sample of the dataset can be committed to the repo under `/scenarios/datasets`
+  3. Scenario should work under the constraints of free-tiers of most cloud providers
+
+To author a scenario, here's a structure you could use:
+  ```
+  - [Introduction](#introduction)
+  - [Technology Stacks](#technology-stacks)
+  - [Pre-requisites](#pre-requisites)
+  - [Running the App](#running-the-app)

```

Comment:
Rather than jumping into Running the App, we could break down the app a bit more (these scenarios can help first time users understand the components of an end-to-end Spark app, and a mass amount of code could be overwhelming). Break into small chunks like "1. Create Spark Session" "2. Read data into DF" "3. Register UDF." Could call overall section "Writing the App," "Understanding the App," "Writing and Running the App," or "Creating a Spark.NET Solution."